### PR TITLE
Set broker to support websocket 

### PR DESCRIPTION
**Pull request by hannahoprchal**
Configuration to make broker container accessible via web-sockets.

### Fixes

- [EXDRHUB-1232](https://issues.redhat.com/browse/EXDRHUB-1232) 

### Checklist

- [ ] Related tests were updated
- [ ] Related documentation was updated
- [ ] OpenAPI file was updated
- [ ] Run `tox`, no tests failed

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -106,11 +106,16 @@ services:
...
       - '5672:5672'
+      - '15674:15674' #webstomp
     healthcheck:
       test: [ "CMD", "rabbitmq-diagnostics", "-q", "ping" ]
       interval: 30s
       retries: 5
       timeout: 30s
+    volumes:
+      - ./config/rabbitmq/rabbitmq.conf:/etc/rabbitmq/rabbitmq.conf
+      - ./config/rabbitmq/enabled_plugins:/etc/rabbitmq/enabled_plugins
```
Comment (guioliveirabh) [inline docker-compose.yml:117]:
```suggestion
      - ./config/rabbitmq/rabbitmq.conf:/etc/rabbitmq/rabbitmq.conf:ro
      - ./config/rabbitmq/enabled_plugins:/etc/rabbitmq/enabled_plugins:ro
```

As these are configuration files, it's safer to set them as read-only.

File: config/rabbitmq/enabled_plugins [hunk 2]
```diff
@@ -0,0 +1 @@
+[rabbitmq_management, rabbitmq_web_stomp, rabbitmq_stomp].
```
Comment (guioliveirabh) [inline config/rabbitmq/enabled_plugins:1]:
```suggestion
[rabbitmq_management, rabbitmq_web_stomp, rabbitmq_stomp].

```
New line at end of file will make next diffs look better.

File: config/rabbitmq/rabbitmq.conf [hunk 3]
```diff
@@ -0,0 +1,3 @@
+#more https://www.rabbitmq.com/configure.html#config-file
+
+web_stomp.tcp.port = 15674
```
Comment (guioliveirabh) [inline config/rabbitmq/rabbitmq.conf:3]:
```suggestion
web_stomp.tcp.port = 15674

```
New line at end of file will make next diffs look better.

File: tox.ini [hunk 4]
```diff
@@ -70,7 +70,7 @@ ignore =
     sonar-project.properties
     tests/**
     tox.ini
-
+    config/rabbitmq/*
```
Comment (guioliveirabh) [inline tox.ini:73]: Actually, you can't ignore these configuration files. You need to include them on the rhub-api package.
You can do that by including them under: [MANIFEST.in](https://github.com/resource-hub-dev/rhub-api/blob/master/MANIFEST.in)

**Review by guioliveirabh — COMMENTED** [review]
Almost everything looks good to me, but I suggest for you to add more people, as I'm mostly dedicated to Beaker, I may be missing something.

File: tox.ini (hunk 4 above)
Comment (fpob) [reply tox.ini:73]: These configs are only for rabbitmq container, not for rhub-api.

**Review by fpob — APPROVED** [review]

**Comment by hannahoprchal** [comment]
Hi @guioliveirabh,
thank you for your suggestions I addressed them into the code and squashed it with other commits.
Feel free to re-review this once you have time.

Thanks,
Hana

**Review by guioliveirabh — APPROVED** [review]

**Pull request merged**